apps/frontend/src/components/proposal/ProposalParticipants.tsx

```diff
@@ -37,39 +46,73 @@ const Participants = ({
   loadingPrincipalInvestigator,
 }: ParticipantsProps) => {
   const [modalOpen, setOpen] = useState(false);
+  const { featuresMap } = useContext(FeatureContext);
+  const isLegacyInviteFlow = featuresMap.get(
+    FeatureId.EMAIL_INVITE_LEGACY
+  )?.isEnabled;
+  const removeUser = (user: BasicUserDetails) => {
+    const newUsers = users.filter((u) => u.id !== user.id);
+    setUsers(newUsers);
+  };
+
+  const openModal = () => {
+    setOpen(true);
+  };
 
   const addUsers = (addedUsers: BasicUserDetails[]) => {
     setUsers([...users, ...addedUsers]);
     setOpen(false);
   };
 
-  const removeUser = (user: BasicUserDetails) => {
-    const newUsers = users.filter((u) => u.id !== user.id);
-    setUsers(newUsers);
+  const handleAddParticipants = (props: {
+    users: BasicUserDetails[];
+    invites: Invite[];
+  }) => {
+    setUsers([...users, ...props.users]);
+    setInvites([...invites, ...props.invites]);
+    setOpen(false);
   };
```

Comment:
Are these two function not doing the same or almost similar function the difference is maybe on invites .Maybe using one function for both legacy and the new flow.

Reply:
Hi, Thanks, I'm keeping it DRY, now. I made addUsers to reference handleParticipants with empty initializer for invites. If we are happy with tne new flow, the fleanup PR will remove the clean up the addUsers()